backend/fms_core/viewsets/sample.py

```diff
@@ -444,4 +443,14 @@ def add_samples_to_study(self, request, pk=None):
         if errors:
             raise ValidationError(errors)
         else:
-            return Response(status=204)
\ No newline at end of file
+            return Response(status=204)
+
+    @action(detail=False, methods=["post"])
+    def sample_ids_by_default_selection_excepted_ids(self, request):
```

Comment:
that is really ugly you are building the backend around your frontend. I am not sure why you would need that ... It also likely destroy any benefit you gain from paging ... I will read on and edit my comment but the ugliness will remain (i don't want to forget to mention it ;).

Reply:
That endpoint is intended to be a compromise in case there are more pages that use PagedItemsTable with selection enabled.

Reply:
It's gone.